app/assets/stylesheets/app/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  border-radius: 2px;
+  background-color: $white;
+  font-family: $main-font;
+
+  &__title {
+    height: 48px;
+    font-weight: 300;
+    line-height: 48px;
+    text-align: center;
+    color: $secondary-color;
+  }
+
+  &__info {
+    font-size: 16px;
```

Comment:
Properties should be ordered color, font-size, line-height, margin, opacity, text-align, width